### PR TITLE
Add solutions for chapter5 tasks

### PR DESCRIPTION
**Pull request by tarmino**

### CODE REVIEW

File: delete_every_third_char.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+"""
+Дана строка. Удалите из нее все символы, чьи индексы делятся на 3.
+"""
+
+S = input()
+for i in range(len(S)):
+	if i%3 != 0:
+		print(S[i], end='')
```
Comment (konstantin2002) [inline delete_every_third_char.py:8]: Please use `print` only after you got a target string.

File: first_and_last_occurences.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+"""
+Дана строка. 
+Если в этой строке буква f встречается только один раз, выведите её индекс. 
+Если она встречается два и более раз, 
+выведите индекс её первого и последнего появления. 
+Если буква f в данной строке не встречается, ничего не выводите.
+"""
+
+S = input()
```
Comment (konstantin2002) [inline first_and_last_occurences.py:9]: Consider using lowercase in var names

File: ladder.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+"""
+По данному натуральному n ≤ 9 выведите лесенку из n ступенек, 
+i-я ступенька состоит из чисел от 1 до i без пробелов.
+"""
+
+n = int(input())
+for i in range(1, n+1):
+	for i in range(1, i+1):
```
Comment (konstantin2002) [inline ladder.py:8]: It would be better not to use the same name for different loop variables

File: first_and_last_occurences.py (hunk 2 above)
Comment (tarmino) [reply first_and_last_occurences.py]: Done

**Review by konstantin2002 — APPROVED** [review]

**Pull request merged**